src/app/business/home/home.component.html

```diff
@@ -118,10 +118,10 @@
                 <input id="name" type="text" name="name" formControlName="name"  pInputText />
             </form-item>
             <form-item label="Type" [required]="true">
-                <p-dropdown [style]="{'min-width':'220px'}" placeholder="Please select" formControlName="type" [options]="allTypes"  ></p-dropdown>
-            </form-item>
-            <form-item label="Region" [required]="true">
-                <input id="region" type="text" name="region"  formControlName="region"  pInputText />
+                <p-dropdown (onChange)="onChange($event)" [style]="{'min-width':'220px'}" placeholder="Please select" formControlName="type" [options]="allTypes"  ></p-dropdown>
+            </form-item>  
+            <form-item id="hide_region" label="Region" [required]="false"  [class.disabled]="isLocalCloud">
```

Comment:
The other storage backend types require the Region field.
Since the Region is not needed only for CEPH type, you should disable the field and set required to false only for this type. For the other types it must still be a required field.

Reply:
Okay, we will modify for other types. Thanks